src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to ingest new metrics in the Observability Platform
```

Comment:
Something like 

description: How to make new metrics available in the Observability Platform in self-service

That would also be 79 chars resolving the Front Matter Problem

Reply:
```suggestion
description: How to make new metrics available in the Observability Platform in self-service
```